tools/tests/test_run_training_job.py

```
@@ -0,0 +1,44 @@
+import sys
+from unittest.mock import MagicMock, patch
+
+# sagemaker is a runtime dependency resolved from the workspace root; it is not
+# available in isolated test environments, so mock its modules before importing
+# the module under test to allow pytest to collect this file.
+sys.modules.setdefault("sagemaker", MagicMock())
+sys.modules.setdefault("sagemaker.estimator", MagicMock())
+sys.modules.setdefault("sagemaker.inputs", MagicMock())
+sys.modules.setdefault("sagemaker.session", MagicMock())
+
+from tools.run_training_job import run_training_job  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)